packages/core/src/blitz-data.tsx

```diff
@@ -0,0 +1,49 @@
+import {getConfig} from "@blitzjs/config"
+import htmlescape from "htmlescape"
+import React from "react"
+import {isClient} from "./utils"
+
+export function _getBlitzRuntimeData() {
+  return {suspenseEnabled: getConfig().experimental?.reactMode !== "legacy"}
+}
+
+export function getBlitzRuntimeData() {
+  if (isClient) {
+    return window.__BLITZ_DATA__
```

Comment:
I can get the tests passing again by changing this line to:

```
    if (!window.__BLITZ_DATA__) {
      window.__BLITZ_DATA__ = _getBlitzRuntimeData()
    }
    return window.__BLITZ_DATA__
```

But that would blow up if it got executed on the client.

Reply:
(It _shouldn't_ get executed on the client, because __BLITZ_DATA__ should already be set on the client...)

Reply:
Maybe it would be better to move this into the tests:

```
window.__BLITZ_DATA__ = _getBlitzRuntimeData()
```

Reply:
I pushed a commit that updates the tests.